packages/atomic/src/components/search/atomic-follow-up-question-list/atomic-follow-up-question-list.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  buildSearchBox,
+  getOrganizationEndpoint,
+  SearchBox,
+  SearchBoxState,
+} from '@coveo/headless';
+import {Component, h, Prop, State} from '@stencil/core';
+import {
+  FollowUpQuestionCandidate,
+  SelectFollowUpQuestionCandidatePayload,
+} from '../../../components/common/follow-up-questions/follow-up-question-list-common';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {Bindings} from '../atomic-search-interface/atomic-search-interface';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-follow-up-question-list',
+  styleUrl: 'atomic-follow-up-question-list.pcss',
+  shadow: true,
+})
+export class AtomicFollowUpQuestionList implements InitializableComponent {
+  @InitializeBindings()
+  public bindings!: Bindings;
+
+  private searchBox!: SearchBox;
+
+  private previousQuery: string = '';
+
+  /**
+   * @internal
+   * The unique identifier of the answer configuration to use to generate the answer.
+   */
+  @Prop()
+  public answerConfigurationId!: string;
```

Comment:
I begin to think that the usage of a prop to handle the answerconfigurationID is not the best one.
This implementation creates another point where we need to pass the `answerConfigurationID`.
Whenever an implementation pass the answerConfigID to this component and not the generated-answer component, we will have weird behavior I think.
And in the end, having to pass the answerConfigurationId to multiple components doesn't seem right IMHO

Reply:
I had the exact same thought on my side. Having the answer configuration ID at the engine level would ensure that all answer-based components remain in-sync.

Reply:
I had the same thought also, glad we are on the same page.

Reply:
Thanks for the suggestion, I love the idea! I'll refrain from implementing it in the current PR as I'm trying to keep its scope as limited as possible, but will create a ticket. We should do it before releasing the feature.